README.md

```diff
@@ -78,7 +78,8 @@ TLDR016     | Label for information link should be spelled exactly `More informa
 TLDR017     | Information link should be surrounded with angle brackets
 TLDR018     | Page should only include a single information link
 TLDR019     | Page should only include a maximum of 8 examples
-TLDR020     | Label for additional notes should be spelled exactly `Note: `
+TLDR020     | Label for additional notes should be spelled exactly `Note:` (with a succeeding whitespace)
```

Comment:
I rebased this PR and merged it (sorry VSCode added previous commits too; removed them in the final commit message). To avoid the Markdown lint error, I replaced the space after `Note:` with a description. But I have left it as it is in the JS file.